### PR TITLE
fix(settings): Use heading for Profile and Profile picture sections

### LINKED ISSUE #41827: [BITV] 9.1.3.1a/6.2 - The headings "Profile picture" and "Profile" have been incorrectly implemented as "label" elements instead of headings - e.g. <label data-v-117e8ad0="">Profile</label>. (1)
e.g. `<label data-v-117e8ad0="">Profile</label>.`

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-be75bbae42-v6-n2

### PR DESCRIPTION
**Pull request by Pytal**
- Fix https://github.com/nextcloud/server/issues/41827

## Summary

- Profile picture has multiple buttons so binding it to one input doesn't make sense
- Profile checkbox already has a `<label>`

> No visual changes

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/components/PersonalInfo/AvatarSection.vue [hunk 1]
```diff
@@ -63,8 +63,7 @@
 					</NcButton>
 				</div>
 				<span>{{ t('settings', 'The file must be a PNG or JPG') }}</span>
-				<input :id="inputId"
-					ref="input"
+				<input ref="input"
```
Comment (susnux) [inline apps/settings/src/components/PersonalInfo/AvatarSection.vue:66]: do we need a visual label here?

**Review by artonge — APPROVED** [review]

File: apps/settings/src/components/PersonalInfo/AvatarSection.vue (hunk 1 above)
Comment (susnux) [reply apps/settings/src/components/PersonalInfo/AvatarSection.vue:66]: cc @ShGKme @JuliaKirschenheuter

Comment (ShGKme) [reply apps/settings/src/components/PersonalInfo/AvatarSection.vue:66]: I don't think so, it is hidden and used in JS programmatically.

**Review by susnux — APPROVED** [review]

**Review by JuliaKirschenheuter — APPROVED** [review]
works!

**Comment by susnux** [comment]
drone unrelated

**Pull request merged**